src/components/Garden/Preferences/Permissions.tsx

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable react/jsx-key */
```

Comment:
Maybe the problem with names don't show up could be related to it. Same if it's not, I'll recommend remove it and set proper keys attributes to react render properly

Reply:
I don't think we really need this line also.